tests/hooks/index.ts

```diff
@@ -0,0 +1,3 @@
+export { useIsFirstRender } from './useIsFirstRender';
```

Comment:
Not sure, about isFirstRender/WhenIsNotFirstRender here. What do you think about:

```suggestion
import { useEffect, useRef } from "react";

// only first render
export const useDidMount = (callback: VoidFunction | (() => VoidFunction)) => {
  useEffect(callback, []);
};

// only on change
export const useDidUpdate = (
  callback: VoidFunction | (() => VoidFunction),
  dependencies: any[],
) => {
  const mountRef = useRef(false);

  useEffect(() => {
    if (!mountRef.current) {
      mountRef.current = true;
      return;
    }

    return callback();
  }, dependencies);
};
```

Reply:
I want to have a separated hooks (userIsFirstRender, useWhenIsNotFirstRender). Now I don`t need for a useDidMount hook, because we don`t use it. I think that the name `useDidUpdate` does not reflect that the hook should be called ignoring the first render

Reply:
Why? **useDidRender** won't be triggered first render.

Reply:
Why do you say about the `useDidRender` hook? There was no anything about one in the previous comment, you suggested using the `useDidUpdate ` hook.

Reply:
Right, **useDidUpdate**. Mistyped in the previous comment. Please checkout  implementation of the **useDidUpdate**.

Reply:
fixed it